code/game/jobs/job/medical.dm

```diff
@@ -36,14 +36,44 @@
 			H.equip_to_slot_or_del(new /obj/item/weapon/storage/box/survival(H.back), slot_in_backpack)
 		return 1
 
+/datum/job/emt
+	title = "Emergency Medical Technician"
+	flag = EMT
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the chief medical officer"
+	selection_color = "#ffeef0"
+	access = list(access_medical, access_morgue, access_chemistry, access_genetics,access_maint_tunnels, access_external_airlocks)
```

Comment:
That line isn't the access line we use here. There are two levels of access in code. minimal_access, and a more expanded access seen on some servers. I simply copied the access line from MD and removed surgery/virology.

+	minimal_access = list(access_medical, access_morgue, access_maint_tunnels, access_external_airlocks)

Is the access list that our EMTs would have with the current configuration settings.